addons/custom-block-text/addon.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Customizable block text style",
+  "description": "Changes the thickness of the text on Scratch blocks and optionally adds a text shadow.",
```

Comment:
Is "Scratch" necessary in this description? I don't think there are any other "blocks" this could possibly refer to, and other addons just call them "blocks".

Reply:
Agreed